e2e/plugin-e2e/plugin-e2e-api-tests/as-admin-user/explorePage.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { expect, test } from '@grafana/plugin-e2e';
+
+import { formatExpectError } from '../errors';
+
+test('query data response should be OK when query is valid', async ({ explorePage, page }) => {
+  await explorePage.datasource.set('gdev-testdata');
```

Comment:
Am I reading this wrong or is page unused?

```suggestion
test('query data response should be OK when query is valid', async ({ explorePage }) => {
  await explorePage.datasource.set('gdev-testdata');
```